index.html

```diff
@@ -4,76 +4,183 @@
+                    <ul class="navbar-nav justify-content-end flex-grow-1 pe-3">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="about.html">À propos</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="project.html">Mes projets</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="contact.html">Contact</a>
+                        </li>
+                        <li class="nav-item dropdown">
+                            <!-- <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" La flèche dropdown et la section dropdown
+                                aria-expanded="false">
+                                Dropdown
+                            </a> -->
+                            <!-- <ul class="dropdown-menu dropdown-menu-dark">
+                                <li><a class="dropdown-item" href="#"></a></li>
+                                <li><a class="dropdown-item" href="#"></a></li>
+                                <li>
+                                    <hr class="dropdown-divider">
+                                </li>
+                                <li><a class="dropdown-item" href="#">Something else here</a></li>
+                            </ul> -->
+                        </li>
+                    </ul>
+
+                </div>
             </div>
         </div>
     </nav>
 
-    <div class="container mt-3">
-        <h1>Bienvenue sur ma page Bootstrap</h1>
-        <p>Cette page est créée avec Bootstrap.</p>
+
+
+    <div class="row">
+        <nav aria-label="breadcrumb">
+            <ol class="breadcrumb">
+                <li class="breadcrumb-item active" aria-current="page">Home</li>
+            </ol>
+        </nav>
     </div>
-    <div class="container">
+
+    <div class="container" data-bs-theme="dark">
+
         <div class="row">
-            <div class="col-4">
-                <div class="card">
-                    <img src="./assets/dog.jpg" class="card-img-top" alt="...">
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">Some quick example text to build on the card title and make up the bulk of
-                            the card's content.</p>
-                        <a href="#" class="btn btn-primary">Go somewhere</a>
-                    </div>
+            <hr class="ligne">
+        </div>
+
+        <div class="row">
+            <img class="lettre" src="assets/markus-spiske-1LLh8k2_YFk-unsplash.jpg" alt="">
+        </div>
+        <br>
```

Comment:
Instead of use `<br>`, could you simply put a `mt-` or `mb-` class in some of your divs?